### PR TITLE
Simplify computations in reporting WF

### PR DESCRIPTION
**Pull request by wm75**
Version 2.0 of column_maker lets us do this.

### CODE REVIEW

**Comment by pvanheus** [comment]
> Hmm, yeah, need to wait for [galaxyproject/usegalaxy-tools#536](https://github.com/galaxyproject/usegalaxy-tools/pull/536)

that now been merged.

File: workflows/sars-cov-2-variant-calling/sars-cov-2-variation-reporting/variation-reporting.ga [hunk 1]
```diff
@@ -45,7 +39,7 @@
             "uuid": "2e5a5b38-c204-45a2-98e2-e113bce5a14b",
             "workflow_outputs": [
                 {
-                    "label": null,
+                    "label": "af_filter_threshold",
```
Comment (mvdbeek) [inline workflows/sars-cov-2-variant-calling/sars-cov-2-variation-reporting/variation-reporting.ga:42]: How did you create an output label here ? I don't think the editor offers this ?
This is where planemo fails. We should be able to give workflow output labels to optional inputs, but if you want the tests to pass for now you'll have to drop this.

**Comment by pvanheus** [comment]
This might also be useful: https://github.com/bgruening/galaxytools/pull/1245

File: workflows/sars-cov-2-variant-calling/sars-cov-2-variation-reporting/variation-reporting.ga (hunk 1 above)
Comment (mvdbeek) [reply workflows/sars-cov-2-variant-calling/sars-cov-2-variation-reporting/variation-reporting.ga:42]: Fixed now in my WIP branch of planemo, that's why the tests have passed.

**Comment by wm75** [comment]
Nice @pvanheus! We can save one more step with that.
@mvdbeek will need toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_find_and_replace/1.1.4 on .org though.

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**